spring-pulsar/src/main/java/org/springframework/pulsar/annotation/PulsarListenerAnnotationBeanPostProcessor.java

```diff
@@ -227,6 +232,7 @@ public void afterSingletonsInstantiated() {
 		}
 
 		if (this.defaultContainerFactoryBeanName != null) {
+			// BONO: Should we check that its not already set on the registrar (like we do above for the endpointRegistry?)
```

Comment:
@sobychacko I just saw these comments I had added in the code that would better be served as PR comments. For now, lets just discuss/answer them here and I will remove the comment from the actual code w/ a commit to this PR once we discuss each (several below as well that fit this same bill).